Description: <change default plymouth theme to blankon>
 0003-default-theme blankon as default theme.
 .
 plymouth (0.9.3-1blankon1) uluwatu; urgency=medium
 .
   * New upstream release
   * 0003-default-theme blankon as default theme
   * debian/control suggests: plymouth-theme-blankon
Author: AnTo SamalonA <ansamsoftdev@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2017-11-06

--- plymouth-0.9.3.orig/src/plymouthd.conf
+++ plymouth-0.9.3/src/plymouthd.conf
@@ -1,3 +1,3 @@
 # Administrator customizations go in this file
 #[Daemon]
-#Theme=fade-in
+Theme=blankon
--- plymouth-0.9.3.orig/src/plymouthd.defaults
+++ plymouth-0.9.3/src/plymouthd.defaults
@@ -1,6 +1,6 @@
 # Distribution defaults. Changes to this file will get overwritten during
 # upgrades.
 [Daemon]
-Theme=spinner
+Theme=blankon
 ShowDelay=5
 DeviceTimeout=5
